agent/testagent.go

```diff
@@ -470,46 +470,46 @@ func TestACLConfig() string {
 }
 
 const (
-	TestDefaultMasterToken      = "d9f05e83-a7ae-47ce-839e-c0d53a68c00a"
-	TestDefaultAgentMasterToken = "bca580d4-db07-4074-b766-48acc9676955'"
+	TestDefaultInitialManagementToken = "d9f05e83-a7ae-47ce-839e-c0d53a68c00a"
+	TestDefaultAgentRecoveryToken     = "bca580d4-db07-4074-b766-48acc9676955'"
```

Comment:
I personally wish we did not use a `Test` prefix for all the testing fixtures. I find it does not read too well. I guess it is particularly important to identify these as "used for testing" since they are in the `agent` package. I wonder if using an `agenttest` package would remove the need for the `Test` prefix.

Not something I think we should change in this PR, just mentioning it as something I've been thinking about.